torch/_subclasses/fake_tensor.py

```diff
@@ -569,6 +569,16 @@ def index_tensor(fake_mode, func, *args, **kwargs):
         return out.to(out_device)
 
 
+# Can take mixed meta/non-meta arguments; the meta registration
+# will roughly do the right thing even when given real devices
+@register_op_impl(aten._embedding_bag.default)
+def embedding_bag(fake_mode, func, *args, **kwargs):
```

Comment:
For my own understanding, this would skip the default FakeTensor behavior that would check that for a consistent device so that the FakeTensor behavior is consistent with that of the meta_registration, is that correct?

Reply:
Yes